net-analyzer/greenbone-security-assistant/greenbone-security-assistant-21.4.4.ebuild

```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake systemd toolchain-funcs
+
+MY_PN="gsad"
+
+DESCRIPTION="Greenbone Security Assistant"
+HOMEPAGE="https://www.greenbone.net/en/ https://github.com/greenbone/gsa"
```

Comment:
Should be now `https://github.com/greenbone/gsad`. I'd also drop the `/en/` part of the first URL.